fastlane/Fastfile

```diff
@@ -48,8 +50,8 @@ fastlane_require 'dotenv'
 USER_ENV_FILE_PATH = File.join(Dir.home, '.wpandroid-env.default')
 Dotenv.load(USER_ENV_FILE_PATH)
 
-ENV[GHHELPER_REPO = 'wordpress-mobile/WordPress-Android']
-ENV['PROJECT_ROOT_FOLDER'] = File.dirname(File.expand_path(__dir__)) + '/'
+ENV.fetch(GHHELPER_REPO = 'wordpress-mobile/WordPress-Android', nil)
```

Comment:
I'm not sure about this change.

I think what the code did (or tried to do) before was setting the `GHHELPER_REPO` value in the environment. Now it's only reading it, I think. 🤔 

By the way, do we even need it in the environment? I thought we removed it a while back. See https://github.com/wordpress-mobile/WordPress-iOS/pull/20072

Reply:
This went away with the latest rebase, so I believe @spencertransier already worked on some refactoring here.

It seems that the old code and the RuboCop updated code weren't doing much 😅

Reply:
> I believe @spencertransier already worked on some refactoring here.

I thought that, too, but couldn't find the PR. I've seen many of those recently because Spencer has been busy across all our apps and assumed I just got confused.